src/components/container/SeverityCalendar/SeverityCalendar.tsx

```diff
@@ -144,6 +145,6 @@ export default function (props: SeverityCalendarProps) {
     if (!data) {
         return <LoadingIndicator />;
```

Comment:
You probably want to add it to the `LoadingIndicator` as well.  Or add a `div` that surrounds them both and add it to that.

Reply:
Thanks, good call out